packages/experimental/Link/src/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { Alert } from 'react-native';
+import { Link } from '..';
+import * as renderer from 'react-test-renderer';
+
+it('Default Link', () => {
```

Comment:
Rerender tests should also be part of the test suite, in case adding all the tests was part of the aim of the PR

Reply:
Rerender tests being part of the test suite is a component v1 requirement. You can see other examples of the tests in other components, like [here](https://github.com/microsoft/fluentui-react-native/blob/3096f1d203c987c89a09fbb37642aa4ad85d3c5a/packages/components/Button/src/Button.test.tsx#L62).